docs/installation.rst

```diff
@@ -101,6 +118,12 @@ If you get an error with ``openssl-dev`` you may have to use ``libressl-dev``.
 Debian/Ubuntu
 ~~~~~~~~~~~~~
 
+.. warning::
+
+    The Rust available by default in most Debian/Ubuntu versions is older than
```

Comment:
18.04 LTS and 20.04 LTS have Rust 1.47 in updates.
Debian testing and upcoming stable have recent Rust, too.